core/auth/identifier.go

```diff
@@ -0,0 +1,51 @@
+package auth
+
+import (
+	"context"
+
+	"flamingo.me/dingo"
+	"flamingo.me/flamingo/v3/framework/config"
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+// BindHTTPBasicAuthIdentifier helper for dingo bindings in projects
+func BindHTTPBasicAuthIdentifier(injector *dingo.Injector) {
+	injector.BindMulti(new(RequestIdentifier)).To(new(HTTPBasicAuthIdentifier))
+}
+
+// HTTPBasicAuthIdentifier identifies users based on HTTP Basic Authentication header
+type HTTPBasicAuthIdentifier struct {
+	users map[string]string
+}
+
+// Inject user configuration
+func (i *HTTPBasicAuthIdentifier) Inject(config *struct {
```

Comment:
identifier contains a sample implementation for http basic auth:
- should we rename file to "basicauth" then? 
-or place this in a subpackage/subfolder "basicauth" to also differenciate it from the other types in that module (that really belongs to the generic auth core)